PixaView/Controller/PixaViewController.swift

```diff
@@ -0,0 +1,276 @@
+        setupCollectionView()
+        fetch(order: order, currentPage: currentPage, filterCategory: category)
+    }
+    
+    //MARK: - view will layout
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        if UIDevice.current.orientation.isLandscape {
+            landscapeLayoutCollectionView()
+        }else if UIDevice.current.orientation.isPortrait{
+            portaitCollectionView()
+        }
+    }
+    
+    
+    // pixa sort button
+    @objc private func pixaSortButton(){
+        dismiss(animated: true, completion: nil)
+        
+        let sortButtonVC = SortButtonVC()
+        sortButtonVC.sortImageDelegate = self
+        self.present(sortButtonVC, animated: true)
+    }
+    
+    
+    //MARK: - actions
+    //filterButtonDidTap
+    @IBAction func filterButtonDidTap(_ sender: UIButton) {
+        
+        guard let filterVC = storyboard?.instantiateViewController(withIdentifier: "FilterViewControllerID") else{return}
+        let filterViewController = filterVC as? TableFilterVC
+        filterViewController?.holder = self
+        filterViewController?.tableViewFilterSelectedDelegate = self
+        
+        
+        let navVC = UINavigationController(rootViewController: filterVC)
+        self.present(navVC, animated: true, completion: nil)
+        filterViewController?.selectedCategories = category
+        
+        
+    } //end of filterButtonDidTap
+    
+    
+    //MARK: <Methods>
+    
+    // collection view setup
+    private func setupCollectionView(){
+        self.collectionView.register(UINib(nibName: PixaViewCell.cellIdentifier, bundle: nil), forCellWithReuseIdentifier: PixaViewCell.cellIdentifier) //assign to cell the Xib file
+        self.collectionView.dataSource = self //the cell need to understand where to get the filling from. add protocol to extension - UICollectionViewDataSource
+        self.collectionView.delegate = self //delegate with collectionView. Subscribe to UICollectionViewDelegate as delegate
+    }
+    
+    
+    //    func saveSortStatus(){
+    //        let filterVC = TableFilterVC()
+    //        //chekedItems = filterVC.selectedCategories
+    //        chekedItems.append(contentsOf: filterVC.selectedCategories)
+    //        print ("main vc contains \(chekedItems)")
+    //    }
+    
+    //MARK: - fetch data
+    func fetch(order: Order, currentPage: Int, filterCategory: [Categories]){
+        let URLSession = URLSession.shared
+        let service: HTTPService = HTTPService(with: URLSessionHttpClient(session: URLSession))
+        //            service.fetchPics(order: order, filterCategory: filterCategory, currentPage: currentPage) { fetchedHits, total in
+        service.fetchPics(order: order, filterCategory: filterCategory, currentPage: currentPage) {result in
+            
+            switch result{
+            case .success(( let hitsRESULT, let total )):
```

Comment:
Could be replaced with 

`case let .success((hitsRESULT, total)):`

Reply:
+